docs/support-bundle-guide.md

```diff
@@ -0,0 +1,183 @@
+The Secret will be used in `authSecret` in the following YAML examples.
+
+```bash
+kubectl create secret generic support-bundle-secret  --from-literal=username='yoursshusername'  --from-literal=password='yoursshpassword'
+```
+
+Then we could apply the following YAML files.
+The first one is to collect support bundle on Antrea Nodes "kube-worker" and "kube-master".
+The second one is to collect support bundle on ExternalNode "vm".
+
+```yaml
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-kube-nodes
+spec:
+  nodes:
+    nodeNames: // empty list means all the Nodes
+      - kube-master
+      - kube-worker
+  expirationMinutes: 10 // expirationMinutes is the requested duration of validity of the SupportBundleCollection. A SupportBundleCollection will be marked as Failed if it does not finish before expiration.
+  fileServer:
+    url: sftp://yourtestdomain.com:22/root/test 
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-vm-external-node
+spec:
+  externalNodes:
+    nodeNames: // empty list means all the ExternalNodes in namespace
+      - vm
+    namespace: vm-ns
+  fileServer:
+    url: yourtestdomain.com:22/root/test //scheme sftp can be omitted
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+You can check the status of SupportBundleCollection by running
+`kubectl get supportbundlecollections NameOfSupportBundleCollection -ojson`. The following example
+shows a successful creation of a SupportBundleCollection. `collectedNodes` and `desiredNodes`
+should match the number of Nodes/ExternalNodes you want to collect support bundle. There is also
+an entry of "Completed" status in the "conditions" list.
+
+```bash
+$ kubectl get supportbundlecollections collect-support-bundle-for-kube-nodes -ojson
+
+...
+ "status": {
+        "collectedNodes": 1,
+        "conditions": [
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "True",
+                "type": "Started"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "BundleCollected"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:35Z",
+                "status": "False",
+                "type": "CollectionFailure"
+            },
+            {
+                "lastTransitionTime": "2022-12-08T06:49:41Z",
+                "status": "True",
+                "type": "Completed"
+            }
+        ],
+        "desiredNodes": 1
+    }
+```
+
+To access the files, SSH into yourtestdomain.com with port 22. Three tarballs,
```

Comment:
To access the files, SSH into yourtestdomain.com with port 22, or scp them to any other storages you expected.